vochain/transaction.go

```diff
@@ -130,7 +130,8 @@ func (app *BaseApplication) AddTx(vtx *VochainTx, commit bool) (*AddTxResponse,
 			if err := app.State.AddProcess(p); err != nil {
 				return nil, fmt.Errorf("newProcess: addProcess: %w", err)
 			}
-			if err := app.State.IncrementAccountProcessIndex(txSender); err != nil {
+			entityAddr := common.BytesToAddress(p.EntityId)
+			if err := app.State.IncrementAccountProcessIndex(entityAddr); err != nil {
```

Comment:
@p4u It is breaking by this line, now that I think more.. Here the AccountProcessIndex will be incremented in different accounts (oracles vs entityID)